config/config.go

```diff
@@ -890,6 +890,9 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("accounts.filesystem.directorypath", "./stored_requests/data/by_id")
 	v.SetDefault("accounts.in_memory_cache.type", "none")
 
+	v.BindEnv("user_sync.external_url")
+	v.BindEnv("user_sync.coop_sync.default")
+
```

Comment:
I am purposefully excluding priority groups since I don't know how to user env variables in Viper for an array of arrays, or even if it's possible.